pkg/agent/agent.go

```diff
@@ -376,6 +374,30 @@ func (i *Initializer) initInterfaceStore() error {
 	return nil
 }
 
+func (i *Initializer) restorePortConfigs() error {
+	ovsCtlClient := ovsctl.NewClient(i.ovsBridge)
+	ovsPorts, err := i.ovsBridgeClient.GetPortList()
+	if err != nil {
+		return fmt.Errorf("failed to list OVS ports: %w", err)
+	}
+	for _, port := range ovsPorts {
```

Comment:
don't we already cache all TrafficControl and IPSec interfaces? I think they can be fetched via `GetInterfacesByType` more efficiently?

Reply:
Done. Thanks for the suggestion!